lib/etc/openturns.conf.in

```diff
@@ -845,6 +845,10 @@
   <AdaptiveDirectionalStratification-DefaultMaximumStratificationDimension value_int="3" />
   <AdaptiveDirectionalStratification-DefaultNumberOfSteps                  value_int="2" />
 
+  <!-- OT::LineSampling parameters -->
+  <LineSampling-DefaultSearchOppositeDirection value_bool="false" />
```

Comment:
Strange. It has been proved that the variance of the antithetic estimator is always less or equal to the one of the unidirectional estimator (eg the antithetic directions are used in Whishart CDF in scipy)

Reply:
do you mean we should change the default ?

Reply:
Yes, I mean: change the default value. If you choose 2N directions at random you get an estimate with a larger variance than if you choose N directions at random and add the N opposite directions in many cases (see eg https://cermics.enpc.fr/~bl/Halmstad/monte-carlo/lecture-2.pdf page 8).